app/Channels/Messages/BotMessage.php

```diff
@@ -113,14 +130,15 @@ public function send(): array
          */
         return [
             'embeds' => [[
-                'color' => $this->color ?? 0,
-                'description' => $this->message,
+                'co2lor' => $this->color ?? 0,
```

Comment:
Think this was a typo

```suggest
'color' => $this->color ?? 0,
```

Reply:
Fixed in 60684353de3ac53f6d5e57f435d46d7d44c2b679